src/lib/gates.ts

```diff
@@ -19,6 +19,24 @@ function rangeCheck0(
   );
 }
 
+/**
+ * the rangeCheck1 gate is used in combination with the rangeCheck0,
+ * for doing a 3x88-bit range check
+ */
```

Comment:
If this is part of the public API you probably want to comment the inputs and output of this gadget.

Reply:
The gates folder is not currently part of the public API